### PR TITLE
check-sof-logger: fix the DSP boot check

### PR DESCRIPTION
**Pull request by kv2019i**
The DSP boot check reload_drivers() must wait longer than 5 seconds as we have many configurations where DSP boot will take more than 60sec (due to e.g. Intel platforms with i915 hardware present but driver not present in the kernel version under test).

Additionally the test case must check whether DSP did boot up after the timeout expires, and fail the test case if it didn't.

### CODE REVIEW

**Review by ujfalusi — APPROVED** [review]
@kv2019i, wow, that's a nice catch (and overlook in the test code)

File: test-case/check-sof-logger.sh [hunk 1]
```diff
@@ -233,14 +233,20 @@ reload_drivers()
 
     # The DSP may unfortunately need multiple retries to boot, see
     # https://github.com/thesofproject/sof/issues/3395
+    # Timeout needs to be 60+sec as platform with GFX present
+    # on hardware but without functional i915 driver, the SOF
+    # probe will require the 60sec i915 probe timeout to expire
+    # before card probe can be completed.
     dlogi "Polling ${CARD_NODE}, waiting for DSP boot..."
-    for i in $(seq 1 5); do
+    for i in $(seq 1 70); do
```
Comment (marc-hb) [inline test-case/check-sof-logger.sh:241]: Can you use `MAX_WAIT_FW_LOADING` here? See #1059 for other use cases

Comment (marc-hb) [reply test-case/check-sof-logger.sh:241]: If you look at the (small) PRs linked from #1059 you will see a new, convenience function `poll_wait_for()` function that can likely replace this loop.

**Review by marc-hb — COMMENTED** [review]
Is this a fix for #01077 ? If yes please mention it in the commit message.

**Review by fredoh9 — COMMENTED** [review]
Good catch!

**Comment by kv2019i** [comment]
Fixed the variable, will try to adopt the poll loop.

**Comment by kv2019i** [comment]
V3 uploaded:
 -  moves to use the poll_wait_for()

**Comment by marc-hb** [comment]
The failure in https://sof-ci.01.org/softestpr/PR1095/build696/devicetest/index.html?model=CML_HEL_RT5682&testcase=check-sof-logger seems unfortunately related. Looking into it.

**Comment by marc-hb** [comment]
Interesting, `rt1011 i2c-10EC1011:00` needs about 10 seconds to boot every time on `cml-hel-rt5682`. Confirmed on two different devices. @plbossart is that a bug?

In any case I will bump the default timeout to 13 seconds.

```
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: snd_sof:ipc3_log_header: sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx: 0x30130000: GLB_TPLG_MSG: PIPE_COMPLETE
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: snd_sof:sof_ipc3_complete_pipeline: sof-audio-pci-intel-cnl 0000:00:1f.3: tplg: complete pipeline PIPELINE.1.SSP0.OUT id 5
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: snd_sof:ipc3_log_header: sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx: 0x30130000: GLB_TPLG_MSG: PIPE_COMPLETE
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: snd_hda_codec_hdmi ehdaudio0D2: ASoC: sink widget AIF1TX overwritten
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: snd_hda_codec_hdmi ehdaudio0D2: ASoC: source widget AIF1RX overwritten
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:00: ADC offset=0x0
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:00: Gain0 offset=0x1ffeea
Sep 06 21:59:16 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:00: Gain1 offset=0x1ffed2
Sep 06 21:59:17 jf-cml-hel-rt5682-01 kernel: usbcore: registered new interface driver snd-usb-audio
Sep 06 21:59:18 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:00: r0 resistance about 7.26 ohm, reg=0x2337F6
Sep 06 21:59:19 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:01: ADC offset=0x0
Sep 06 21:59:19 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:01: Gain0 offset=0x1bf
Sep 06 21:59:19 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:01: Gain1 offset=0x196
Sep 06 21:59:21 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:01: r0 resistance about 6.89 ohm, reg=0x2521A7
Sep 06 21:59:21 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:02: ADC offset=0x0
Sep 06 21:59:21 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:02: Gain0 offset=0x1ffca4
Sep 06 21:59:21 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:02: Gain1 offset=0x1ffd97
Sep 06 21:59:23 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:02: r0 resistance about 5.92 ohm, reg=0x2B2D08
Sep 06 21:59:23 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:03: ADC offset=0x0
Sep 06 21:59:23 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:03: Gain0 offset=0x276
Sep 06 21:59:23 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:03: Gain1 offset=0x2d2
Sep 06 21:59:25 jf-cml-hel-rt5682-01 kernel: rt1011 i2c-10EC1011:03: r0 resistance about 6.16 ohm, reg=0x297CAC
Sep 06 21:59:26 jf-cml-hel-rt5682-01 kernel: snd_sof:sof_pcm_new: sof-audio-pci-intel-cnl 0000:00:1f.3: creating new PCM DMIC16kHz
Sep 06 21:59:26 jf-cml-hel-rt5682-01 kernel: snd_sof:sof_pcm_new: sof-audio-pci-intel-cnl 0000:00:1f.3: spcm: allocate Sound Trigger Capture 8 capture DMA buffer size 0x83400 max 0x138800
Sep 06 21:59:26 jf-cml-hel-rt5682-01 kernel: snd_sof:sof_pcm_new: sof-audio-pci-intel-cnl 0000:00:1f.3: creating new PCM Port1
Sep 06 21:59:26 jf-cml-hel-rt5682-01 kernel: snd_sof:sof_pcm_new: sof-audio-pci-intel-cnl 0000:00:1f.3: spcm: allocate Passthrough Playback 0 playback DMA buffer size 0x10000 max 0x10000
```

**Comment by marc-hb** [comment]
SOFCI TEST

**Pull request merged**